rllab/dynamics_randomization/variation.py

```diff
@@ -0,0 +1,204 @@
+from enum import Enum
+
+
+class VariationMethod(Enum):
```

Comment:
You can just call this `Method` since its use is clear inside the package.

Reply:
Done